compiler/noirc_frontend/src/elaborator/comptime.rs

```diff
@@ -374,12 +386,15 @@ impl<'context> Elaborator<'context> {
     /// Returns any new items generated by attributes.
     pub(super) fn run_attributes(
         &mut self,
+        module_attributes: &Vec<ModuleAttribute>,
         traits: &BTreeMap<TraitId, UnresolvedTrait>,
         types: &BTreeMap<StructId, UnresolvedStruct>,
         functions: &[UnresolvedFunctions],
     ) -> CollectedItems {
         let mut generated_items = CollectedItems::default();
 
+        self.run_attributes_on_modules(module_attributes, &mut generated_items);
```

Comment:
Request from Grego: Can we run module attributes _after_ each other attribute type instead? So after functions in this case

Also: since the ordering we run them in matches the ordering of parameters in this function, lets update that too.